code/_onclick/click.dm

```diff
@@ -29,6 +29,16 @@
 		adj += S.nextmove_adjust()
 	next_move = world.time + ((num + adj)*mod)
 
+/mob/living/carbon/changeNext_move(num)
+	var/mod = next_move_modifier
+	var/adj = next_move_adjust
+	for(var/i in status_effects)
+		var/datum/status_effect/S = i
+		mod *= S.nextmove_modifier()
+		adj += S.nextmove_adjust()
+
+	next_move = world.time + ((num + adj)*mod)

```

Comment:
This is now the same as `/mob/living/changeNext_move(num)` correct?

Reply:
Oh shoot, you're right